test/integration/protocol_integration_test.h

```diff
@@ -18,6 +18,10 @@ class ProtocolIntegrationTest : public HttpProtocolIntegrationTest {
              << "Please use DownstreamProtocolIntegrationTest instead of ProtocolIntegrationTest.";
     }
   }
+
+  // TODO(asraa) remove this when flipping fuzzers on.
+  Filesystem::ScopedUseMemfiles use_memfiles_{true};
```

Comment:
temp TODO for yourself to not submit until you reset to false?

Reply:
I prefer making sure it continued to work, hence leaving one test using memfiles until Asra turns up the fuzzers.

Reply:
Ah and indeed it looks like the scoped version plays less with coverage.  Feel free to hold off on review for a bit - I'll ping when CI is happy